gumby/sync.py

```diff
@@ -337,39 +365,68 @@ def __init__(self, vars):
         self.vars = vars
         self.all_vars = {}
         self.time_offset = None
+        self.scenario_runner = ScenarioRunner()
+        self.scenario_runner.preprocessor_callbacks["module"] = self._preproc_module
+        self.loaded_experiment_module_classes = []
+        self.experiment_modules = []
+        self._stats_file = None
+        if not hasattr(self, 'scenario_file'):
```

Comment:
It's a relic. Inheritors of ExperimentClient would set this variable to indicate the scenario file they'd want to run. I kept it there at first and apprently forgot about it.